docs/development/affiliated-packages.rst

```diff
@@ -660,3 +660,21 @@ by ``CHANGES.md`` in the instructions.
       document.body.innerHTML = document.body.innerHTML.replace(/&lt;packagename&gt;/g, packagename);
     }
     </script>
+
+
+Modifications for a beta/release candidate release
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+   For major releases we do beta and/or release candidates to have a chance to
```

Comment:
Maybe you want to reword this a bit, as this docs doesn't speak about major releases previously.